README.md

```diff
@@ -62,6 +62,24 @@ At the moment, only source installation is supported. Use Ubuntu Focal.
     colcon build
     ```
 
+### Using docker
+#### Requirements
+1. Install Docker using [installation instructions.](https://docs.docker.com/engine/install/ubuntu/)
+1. Complete the [Linux Postinstall steps](https://docs.docker.com/engine/install/linux-postinstall/) to allow you to manage Docker as a non-root user.
+1. Install `rocker` by `sudo apt-get install python3-rocker`.
+
+#### Usage
+Clone the buoy_entrypoint repository to download the latest Dockerfile.
+    ```
+    git clone https://github.com/osrf/buoy_entrypoint.git
+    cd ~/buoy_entrypoint/docker/
+    ./build.bash buoy
+    ./run.bash [-d|s] buoy:latest
+    ```
+where `./run.bash` option:
```

Comment:
It looks like the run script is just a wrapper around `rocker`. Instead of hiding that from the users, we could consider listing the `rocker` options directly here. This way, users know what's going on and may even use different arguments according to their needs. For example:

```
To run simulation, we recommend you run rocker with the following arguments:

    rocker --devices /dev/dri/ --dev-helpers --nvidia --x11 --user --git

For development, you may want to mount your home directory by adding the `--home` option, i.e.:

    rocker --devices /dev/dri/ --dev-helpers --nvidia --x11 --user --home --git

See the [rocker](https://github.com/osrf/rocker) documentation for additional options.
```

`rocker` is already a wrapper around `docker`. I think we could avoid adding another wrapper with a different set of options.

Reply:
That's fair. Thanks for the note 👍🏽